opencog/persist/file/fast_load.cc

```diff
@@ -203,21 +202,16 @@ static Handle recursive_parse(const std::string& s,
         "Got a Value, not supported: " + s);
 }
 
-/// load_file -- load the given file into the given AtomSpace.
-void opencog::load_file(std::string fname, AtomSpace& as)
-{
-    std::ifstream f(fname);
-    if (not f.is_open())
-       throw std::runtime_error("Cannot find file >>" + fname + "<<");
-
+Handle parseStream(std::istream& in, AtomSpace& as){
```

Comment:
OK, so you made this not be `static` but you didn't put it in the header file ...